vendors/swedbank-pay-sdk-php/tests/bootstrap.php

```diff
@@ -16,9 +16,10 @@
 
 require_once __DIR__ . '/TestCase.php';
 
-if (getenv('MERCHANT_TOKEN') && getenv('PAYEE_ID')) {
+if (getenv('MERCHANT_TOKEN') && getenv('PAYEE_ID') && getenv('VERSION')) {
     define('MERCHANT_TOKEN', getenv('MERCHANT_TOKEN'));
-    define('PAYEE_ID', getenv('PAYEE_ID'));        
+    define('PAYEE_ID', getenv('PAYEE_ID'));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function getenv() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)